runTests.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+#
+
+#export SYMFONY_VERSION="~2.7.0"
+SUPPORTED_SYMFONY_VERSIONS=('~2.3.0' '~2.5.0' '~2.6.0' '~2.7.0' '2.8.x-dev')
+export DOCTRINE_VERSION=">=2.2.3,<2.5-dev"
+
+sudo apt-get -qq update
+sudo apt-get install -y php5 php5-mongo php5-sqlite
+php --version
+
+# Install/upgrade composer.phar
+curl -sS https://getcomposer.org/installer | php
+php composer.phar --version
+
+# Save a copy of the current composer.json file so we can restore it later
+cp composer.json composer.json.bck
+
+# Clear temp folder before the tests run
+sudo rm -rf /tmp/VichUploaderBundle/
```

Comment:
`sudo` isn't necessary (and can be scary when you launch the command without reading its sources :D)